src/parser.jl

```diff
@@ -67,19 +53,13 @@ function parsetagvalues(s)
     for line in eachline(s)
         line = strip(removecomments(line))
         m = match(r_stanza, line)
-        if m != nothing
-            return vals, m.captures[1]
-        end
+        (m !== nothing) && return vals, m.captures[1]
 
         isempty(line) && continue
 
         tag, value, ok = tagvalue(line)
         ok || error("cannot find a tag (position: $(position(s))), empty: $(isempty(line)), line: `$(line)`")
-        if haskey(vals, tag)
-            push!(vals[tag], value)
-        else
-            vals[tag] = [value]
-        end
+        push!(get!(()->Vector{String}(), vals, tag), value)
```

Comment:
The shorter version would invariably create `String[]` even when `vals[tag]`already exists. "Lambda" version (as the old code) creates an array only when necessary (but it's more effective than the old code because the key is searched only once).